### ISSUE TITLE
Could anyone share their fine-tuning parameters for the depth?

### ISSUE THREAD

**Issue by tresym** [tinghuiz/SfMLearner#78]
As the author mentioned in the paper the depth training for kitti is fine-tuned based on the Cityscapes trained model. Could @tinghuiz please share the parameters about fine-tuning also? I am sure it would also help a lot of other users!

Thank you a lot!

**Comment by tinghuiz** [comment]
I think I used a 10x smaller learning rate when starting the fine-tuning. Other than that, everything should be the same.

**Comment by B-Betty** [comment]
10x smaller learning rate? not 10x smaller smooth_weight in fine-tune ?

**Comment by tinghuiz** [comment]
10x smaller learning rate. The smooth_weight is the same.
